it/v1/endpoints/CreateSelfEmploymentPeriodicControllerISpec.scala

```diff
@@ -0,0 +1,875 @@
+         |      },
+         |      "premisesRunningCosts": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "maintenanceCosts": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "adminCosts": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "advertisingCosts": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "businessEntertainmentCosts": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "interestOnLoans": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "financialCharges": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "badDebt": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "professionalFees": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "depreciation": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      },
+         |      "other": {
+         |         "amount": 500.25,
+         |         "disallowableAmount": 500.25
+         |      }
+         |   }
+         |}
+         |""".stripMargin
+    )
+
+    val responseBody: JsValue = Json.parse(
+      s"""
+         |{
+         |  "periodId": "2017-01-25_2017-04-25",
+         |  "links": [
+         |    {
+         |      "href": "/individuals/business/self-employment/$nino/$businessId/period/$periodId",
+         |      "method": "GET",
+         |      "rel": "self"
+         |    }
+         |  ]
+         |}
+         |""".stripMargin)
+
+    def uri: String = s"/$nino/$businessId/period"
+
+    def desUri: String = s"/income-tax/nino/$nino/self-employments/$businessId/periodic-summaries"
+
+    def setupStubs(): StubMapping
+
+    def request(): WSRequest = {
+      setupStubs()
+      buildRequest(uri)
+        .withHttpHeaders((ACCEPT, "application/vnd.hmrc.1.0+json"))
+    }
+
+    def errorBody(code: String): String =
+      s"""
+         |      {
+         |        "code": "$code",
+         |        "reason": "des message"
+         |      }
+    """.stripMargin
+  }
+
+  "Calling the create endpoint" should {
+
+    "return a 200 status code" when {
+
+      "any valid request is made" in new Test {
+
+        override def setupStubs(): StubMapping = {
+          AuthStub.authorised()
+          MtdIdLookupStub.ninoFound(nino)
+          DesStub.onSuccess(DesStub.POST, desUri, NO_CONTENT, JsObject.empty)
```

Comment:
```suggestion
          DesStub.onSuccess(DesStub.POST, desUri, OK, desJson)
```

Make sure desJson is representative of what DES will return